### DISCUSSION TITLE
Unable to run behind reverse proxy

### DISCUSSION THREAD

**Discussion by L0ric0** [jcorporation/myMPD#563] [Q&A] [answered]
Trying to set up mympd behind a reverse proxy with the new config.
When setting
```
#/var/lib/mympd/config/http_host
127.0.0.1
```
and
```
#/var/lib/mympd/config/http_port
8080
```
and configuring nginx to forward requests to `domain.tld/mympd` to `127.0.0.1:8080` the browser is redirected to `https://127.0.0.1` which obviously is wrong.

Could the ability to run mympd behind a reverseproxy be added again?

**Comment by jcorporation** [comment, marked as answer]
You should disable ssl. If ssl is enabledd myMPD redirects http to https. 

https://jcorporation.github.io/myMPD/additional-topics/behind-a-reverse-proxy
